tests/microos/networkmanager.pm

```diff
@@ -88,23 +88,6 @@ true'
     ($RcManager, $mode) = dns_mgr();
     die 'wrong DNS-Manager is currently used for dnsmasq' if ($RcManager !~ /symlink/ || $mode !~ /dnsmasq/);
     ping_check;
-    # systemd-resolved
```

Comment:
If we are skipping this on ALP, I would also encapsulate it in an `is_ALP` if-block to avoid reduced coverage on SLE Micro.

Please guard the whole block within an `if (!is_ALP)` block.

Reply:
Good comments, let me do